app/views/schools/advice/electricity_recent_changes/_insights.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= component 'notice', status: :neutral do |c| %>
+  <% c.with_link { link_to t('advice_pages.electricity_recent_changes.insights.link'), learn_more_school_advice_electricity_recent_changes_path(@school) } %>
+  <%= t('advice_pages.electricity_recent_changes.insights.summary') %>.
+<% end %>
+
+<%= render 'schools/advice/section_title', section_id: 'your_recent_electricity_use', section_title: t('advice_pages.electricity_recent_changes.insights.your_recent_electricity_use.title') %>
+
+<table class="mt-2 table table-sm advice-table">
```

Comment:
Given that the comparison is for last 2 weeks, if we have no recent data then I don't think we can show the detail in the table.

We already show a standard notice, but if there's no recent data, I suggest you exclude the detail for each row. See what I did in the baseload insights page.